tests/js/test-declarations/jest-each.js

```diff
@@ -60,3 +60,18 @@ test.each([
     expect(Number(a)).toBe(b);
   }
 );
+
+test.each`
```

Comment:
Please extract these tests to a separate file. When test files get long, their snapshots are difficult to read.